src/main/webapp/app/application/application-detail-for-applicant/application-detail-for-applicant.component.html

```diff
@@ -11,6 +11,13 @@
           ></h1>
           <jhi-application-state-for-applicants [state]="application.applicationState" />
         </div>
+        <jhi-button
+          [label]="'entity.action.pdf'"
+          [shouldTranslate]="true"
+          severity="secondary"
+          [variant]="'outlined'"
+          (click)="onDownloadPDF()"
+        />
         @if (['SAVED'].includes(application.applicationState)) {
           <jhi-button [label]="'entity.applicationOverview.actions.update'" [shouldTranslate]="true" (click)="onUpdateApplication()" />
         }
```

Comment:
Can you reduce the padding between the buttons

```suggestion
        <div class="flex gap-2">
          <jhi-button
            [label]="'entity.action.pdf'"
            [shouldTranslate]="true"
            severity="secondary"
            [variant]="'outlined'"
            (click)="onDownloadPDF()"
          />
          @if (['SAVED'].includes(application.applicationState)) {
            <jhi-button [label]="'entity.applicationOverview.actions.update'" [shouldTranslate]="true"   (click)="onUpdateApplication()" />
          }
       </div>
```